src/app/openmrs-api/order-resource.service.ts

```diff
@@ -64,6 +102,19 @@ export class OrderResourceService {
     }));
   }
 
+  saveDrugOrder(payload) {
+    if (payload) {
```

Comment:
What purpose does the if statement serve? Looks to me like a debug block of code.